students/MikeShand/lesson03/mailroom.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+
+
+donors=[['Andy', [10.00]],['Bill',[15.00, 25.00]],['Chuck',[20.00,30.00,40.00]]]
+
+def donor_list():
+    donor_list=[]
+    for x in donors:
+        print(x[0])
+    thank_you()
+
+
+def thank_you():
+    print('Please enter the donor name\n (Type "list" for a list of current donor names)\n Press "q" to return to console')
+    cd=input(':')
+
+    if cd.lower()=='list':
+        donor_list()
+
+    elif cd.lower()=='q':
+        console()   

```

Comment:
return here instead of calling console (see my other comment about program flow)